File: fase06/ex01/ateZero.js
function ateZero(num) {
    let arr = [];
    if (num > 0) {
        while (num >= 0) {
            arr.push(num--);
        }
        arr.reverse();
    } else {
        while (num <= 0) {
            arr.push(num++);
        }
    }
    return arr
}

File: fase06/ex01/ateZero.py
def ateZero(nbr):
    arr = []
    if nbr > 0:
        i = 0
        while i <= nbr:
            arr.append(i)
            i +=1
    else:
        while nbr <= 0:
            arr.append(nbr)
            nbr += 1
    return arr
